src/fontra/client/core/glyph-lines.js

```diff
@@ -54,14 +54,23 @@ async function glyphNamesFromText(text, characterMap, glyphMap) {
           if (baseGlyphName === charString) {
             // The base glyph name is a single character, let's see if there's
             // a glyph name associated with that character
-            let properBaseGlyphName = characterMap[baseCharCode];
-            if (!properBaseGlyphName) {
-              properBaseGlyphName = await getSuggestedGlyphName(baseCharCode);
-            }
-            if (properBaseGlyphName) {
-              glyphName = properBaseGlyphName + extension;
-              if (!extension) {
-                char = charString;
+            if (baseGlyphName.search(/[^a-zA-Z]+/) === -1) {
```

Comment:
I don't think this is correct. You are searching for any non-A-Z and non-a-z chars, but that is not the right criterium.

I think the preceding `if (baseGlyphName === charString)` condition should be changed to something like:

```js
    if (baseGlyphName === charString && !isPlainLatinLetter(baseGlyphName) ) {
```

Where `isPlainLatinLetter(glyphName)` checks whether the name is exactly one character, matching `[A-Za-z]`. For example (untested):

```js
function isPlainLatinLetter(glyphName) {
  return glyphName.match(/^[A-Za-z]$/)
}
```

Then the rest of the code can remain unchanged.

Reply:
Where should we store the new function `isPlainLatinLetter`?

Reply:
Within utils?

Reply:
Or at the end of the module.

Reply:
Please review: https://github.com/googlefonts/fontra/pull/1100/commits/28feb5d3fb22425aaec603ea3a398236b7500e42